payments.md

```diff
@@ -0,0 +1,161 @@
+## BusinessPayment
+
+## Payment collection [/payments]
+payments endpoint allows you to make a Payment anonymously or by logging in for Britishgas business customers. Responses will be returned in JSON format only.
+
+### BusinessPayment
+Payment as a resource represents following information
+
+| Property | Type | As request | As response | Description |
+| :-------------------- | :---------- | :-------------------- | :-------------------- | ------------------------------------------------------------ |
+| id | String | `Mandatory` | `Returned always` | It is the payment document number considered as identifier for payment transaction.|
+| card | String | `Manadatory` | `Returned always` | It is the card identifier used for making payment. |
+| postcode | String | 'Optional' | 'Returned always' | It is the post code which is required for making anonymous payment. |
+| amount | int | `Mandatory` | `Returned always` | It is the amount the customer wish to pay. |
+| contractAccount | String | `Returned always` | It is the account for which the customer is making a payment. |
```

Comment:
No camel casing for attribute names. You can use something like contract-account-number.